.ci/functionalTests.groovy

```diff
@@ -98,6 +100,7 @@ pipeline {
               junit(allowEmptyResults: true, keepLongStdio: true, testResults: "${BASE_DIR}/outputs/junit-*.xml")
               archiveArtifacts allowEmptyArchive: true, artifacts: "${BASE_DIR}/outputs/junit-*"
               githubCheckNotify(currentBuild.currentResult == 'SUCCESS' ? 'SUCCESS' : 'FAILURE')
+              sh script: 'make -C metricbeat-tests shutdown-elastic-stack', label: 'Shutdown runtime dependencies'
```

Comment:
I just realized if the make uses the `go` command, then it might be required to create a wrapper script to prepare the context for the given `${GO_VERSION}`. and replace the call for something like:
`.ci/scripts/shutdown-stack.sh "${GO_VERSION}"`

Reply:
[gvm](https://github.com/andrewkroh/gvm/releases/tag/v0.2.1) can help to get specific Go versions and configure the environment variables for it.

Reply:
No, it does not use Go directly: it calls the go binary instead